tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -531,9 +531,11 @@ class ExecutionTest {
                        L"Table:ShaderOpArithTable.xml#PackUnpackOpTable")
   END_TEST_METHOD()
 
+#ifdef D3D12_EXPERIMENTAL_WAVE_MATRIX
```

Comment:
I thought the idea was just to replace the `#if !defined(D3D12_SDK_VERSION) || (D3D12_SDK_VERSION < 613)` with `#ifndef D3D12_EXPERIMENTAL_WAVE_MATRIX`, rather than disable the whole test based on the lack of this define.